server/AuthController.js

```diff
@@ -12,6 +13,42 @@ module.exports = {
     res.set(JWT.renewTokenHeader, token);
     res.status(200).send({ token })
   },
+  impersonate: (_grabUser, id='id') => async (req, res) => {
```

Comment:
We generally don't prefix variables with `_`. Also, we should be consistent in our naming in this library - in other places we called it `findOne` instead of `grabUser` (which also sounds like an HR violation :P ). Let's keep a consistent name everywhere